mmv1/products/memcache/Instance.yaml

```
@@ -37,7 +37,8 @@ async:
     base_url: '{{op_id}}'
   result:
     resource_inside_response: true
-custom_code:
+custom_code: !ruby/object:Provider::Terraform::CustomCode
```

This is no longer ruby-based yaml, so this is not necessary.

```suggestion
custom_code:
```